NEWS

```diff
@@ -37,6 +37,8 @@ patent data files, and the related IMGT parser was updated to cope with
 IPD-IMGT/HLA database files after release v3.16.0 when their ID line changed.
 The GenBank output now uses colon space to match current NCBI DBLINK lines.
 
+The Bio.Affy package supports Affymetrix version 4 of the CEL file format, in addition to version 3.
```

Comment:
Again, line wrap as:

```
The Bio.Affy package supports Affymetrix version 4 of the CEL file format, in
addition to version 3.
```

NOTE to self: Can we add a quick one line Unix command to the TravisCI/Tox settings along side running PEP8 to check this file?